iothub/device/src/Transport/AmqpIot/AmqpIotTransport.cs

```diff
@@ -75,17 +71,26 @@ internal async Task<TransportBase> InitializeAsync(CancellationToken cancellatio
                 Logging.Enter(this, nameof(InitializeAsync));
 
             TransportBase transport;
+            AmqpTransportInitiator amqpTransportInitiator;
 
             switch (_amqpTransportSettings.Protocol)
             {
                 case IotHubClientTransportProtocol.Tcp:
-                    var amqpTransportInitiator = new AmqpTransportInitiator(_amqpSettings, _tlsTransportSettings);
+                    amqpTransportInitiator = new AmqpTransportInitiator(_amqpSettings, _tlsTransportSettings);
```

Comment:
Can this call be factored outside of the switch? We have the same call on line 93.

Also, this one is missing the ConfigureAwait(false) call.